js/modules/k6/execution/execution.go

```diff
@@ -71,8 +72,9 @@ func (*RootModule) NewModuleInstance(m modules.InstanceCore) modules.Instance {
 			common.Throw(rt, err)
 		}
 	}
-	defProp("scenario", mi.newScenarioInfo)
 	defProp("instance", mi.newInstanceInfo)
+	defProp("scenario", mi.newScenarioInfo)
+	defProp("test", mi.newTestInfo)
```

Comment:
I exposed this as `exec.test.abort()` as it was before the `test`/`instance` rename. I think it makes sense to have this under the global "test" property, since it will abort the entire test and not just an instance in the cloud.